db/migrate/20231106151705_default_remixed_project_origin.rb

```diff
@@ -0,0 +1,20 @@
+class DefaultRemixedProjectOrigin < ActiveRecord::Migration[7.0]
+  def up
+    if Rails.env.development?
+      remix_origin = 'http://localhost:3010'
+    elsif Rails.env.test?
+      remix_origin = 'staging-editor.raspberrypi.org'
```

Comment:
I don't think this will have the desired effect; looks like `Rails.env` is `production` on staging (which probably needs fixing?)

```
heroku run --app editor-api-staging  rails c
Running rails c on ⬢ editor-api-staging... up, run.3222 (Standard-1X)
[Scout] [11/09/23 15:32:53 +0000 run.3222 (2)] INFO : No configuration file loaded, and no configuration found in ENV. For assistance configuring Scout, visit https://docs.scoutapm.com/#ruby-configuration-options
[Scout] [11/09/23 15:32:53 +0000 run.3222 (2)] INFO : Scout Agent [5.3.5] Initialized
[Scout] [11/09/23 15:32:53 +0000 run.3222 (2)] INFO : Not Loading Instruments
[Scout] [11/09/23 15:32:53 +0000 run.3222 (2)] INFO : Scout Agent [5.3.5] Installed
[Scout] [11/09/23 15:32:53 +0000 run.3222 (2)] INFO : Monitoring isn't enabled for the [production] environment.
[Scout] [11/09/23 15:32:53 +0000 run.3222 (2)] INFO : Agent attempting to load in interactive mode.
[Scout] [11/09/23 15:32:53 +0000 run.3222 (2)] INFO : Deferring agent start. Standing by for first request
Loading production environment (Rails 7.0.4.2)
irb(main):001:0> Rails.env
=> "production"
```

Reply:
Oh dear... I didn't know that

Reply:
Yeah I remember fixing this on projects-admin not long ago actually:

```
heroku run --app learning-admin-staging rails c
Running rails c on ⬢ learning-admin-staging... up, run.5652 (Standard-1X)
<snip>
Loading staging environment (Rails 6.1.5.1)
irb(main):001:0> Rails.env
=> "staging"

Reply:
I think we should just fix the env on staging @sra405 ? Then this can work if we just use `Rails.env.staging?`

Reply:
Hmm yeah, that makes sense to me 👍

Reply:
these are all set [in terraform](https://github.com/RaspberryPiFoundation/terraform/blob/3be8b8ac976aed7675dc8e2b46d0b37d7580ca0e/projects/editor-api/nonprod/terraform.tfvars#L22-L23) so we'd need to make the change there. I'm happy either way with this one but it makes sense to create a staging environment if we slightly different behaviour now 👍